src/burn/engine/plan.cpp

```diff
@@ -853,6 +877,11 @@ static HRESULT ProcessPackage(
             hr = PlanLayoutPackage(pPlan, pPackage, wzLayoutDirectory);
             ExitOnFailure(hr, "Failed to plan layout package.");
         }
+        else if (BOOTSTRAPPER_ACTION_CACHE == pPlan->action)
```

Comment:
Not sure I follow. You're saying that if `BURN_PLAN::action` is `_CACHE`, then just set the default *package* request state to `_CACHE` and let the rest flow through naturally?